crates/stats-agg/src/stats1d.rs

```diff
@@ -316,15 +354,28 @@ mod tests {
     use super::*;
     use approx::assert_relative_eq;
 
+    fn tf(f: f64) -> TwoFloat {
+        TwoFloat::new_add(f, 0.0)
+    }
+
     #[track_caller]
-    fn assert_close_enough(s1: &StatsSummary1D, s2: &StatsSummary1D) {
+    fn assert_close_enough(s1: &StatsSummary1D<f64>, s2: &StatsSummary1D<f64>) {
         assert_eq!(s1.n, s2.n);
         assert_relative_eq!(s1.sx, s2.sx);
         assert_relative_eq!(s1.sx2, s2.sx2);
         assert_relative_eq!(s1.sx3, s2.sx3);
         assert_relative_eq!(s1.sx4, s2.sx4);
     }
 
+    #[track_caller]
+    fn assert_close_enough_tf(s1: &StatsSummary1D<TwoFloat>, s2: &StatsSummary1D<TwoFloat>) {
+        assert_eq!(s1.n, s2.n);
+        assert!((s1.sx - s2.sx).abs() < 0.0001);
```

Comment:
Why are we reducing the precision o f this comparison so much?

Reply:
I made the assertion more precise.